GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifyAccountDetail.m

```diff
@@ -21,21 +21,21 @@
 
 @implementation GIDVerifyAccountDetail
 
-#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+#if TARGET_OS_IOS
```

Comment:
This can be placed above the `@implementation`.

E.g., https://github.com/google/GoogleSignIn-iOS/blob/fac-eap/GoogleSignIn/Sources/GIDAppCheck/Implementations/GIDAppCheck.h

Reply:
Done.